src/manage.c

```diff
@@ -6345,7 +6419,7 @@ gvm_migrate_secinfo (int feed_type)
       return -1;
     }
 
-  ret = feed_lockfile_lock (&lockfile);
```

Comment:
What about the old function? Is is still used or can it be removed?

Reply:
The old function is still used in the automatic sync that has essentially its own retry loop.
I've also changed `feed_lockfile_lock_timeout` to use it now so writing the timestamp doesn't have to be duplicated.